config/settings.yml

```diff
@@ -137,6 +137,10 @@ sys_api_key:
   default: ''
 repository_authentication_caching_enabled:
   default: 1
+repositories_automatic_managed_repos:
+  default: 0
```

Comment:
That is correct. It will not have any effect, besides making it clear that you intend to use this as a boolean setting :)